.github/ISSUE_TEMPLATE/tracking-issue.md

```diff
@@ -7,13 +7,36 @@ labels: management/tracking, status/proposed
 
 ## Description
 
-<!-- Short description of the proposed feature -->
+Short description of the proposed feature.
 
-## Working Backwards
+## Roles
 
-<!-- Present this feature/change to users as if it was already been implemented -->
+| Role                | User
+|---------------------|------------------------------
+| Proposed by         | @alias
+| Author              | @alias
```

Comment:
Strongly suggest opening up the process to **collaborative** editing. It feels lonely right now.

```suggestion
| Author(s)           | @alias, @alias, @alias
```